sierra_adapter/sierra_item_merger/src/test/scala/uk/ac/wellcome/platform/sierra_item_merger/services/SierraItemMergerUpdaterServiceTest.scala

```diff
@@ -103,39 +117,48 @@ class SierraItemMergerUpdaterServiceTest
           bibIds = bibIds
         ),
         "i999" -> anotherItem
-      ),
-      version = 1
+      )
     )
 
-    Scanamo.put(dynamoDbClient)(tableName)(newRecord)
-
-    whenReady(sierraUpdaterService.update(itemRecord)) { _ =>
-      val expectedNewSierraRecord =
-        SierraTransformable(
-          sourceId = bibIdNotExisting,
-          maybeBibData = None,
-          itemData = Map(itemRecord.id -> itemRecord),
-          version = 1
+    val f2 = hybridStore.updateRecord[SierraTransformable](newRecord)
+    whenReady(Future.sequence(List(f1, f2))) { _ =>
```

Comment:
This doesn't enforce one happening after the other. If that's what you want you should use `f1.flatMap(_ => f2)`

Reply:
the order doesn't matter, just that they both happen before the next whenReady